src/OrchardCore.Themes/TheAdmin/Views/Layout.cshtml

```diff
@@ -41,6 +41,7 @@
                 </div>
                 <div class="d-flex align-items-center">
                     @await RenderSectionAsync("NavbarTop", required: false)
+                    @await DisplayAsync(await New.CulturePicker())
```

Comment:
Need some way to show it if the `CulturePicker` feature is enabled, related to https://github.com/OrchardCMS/OrchardCore/discussions/12014

Reply:
You can always add a setting like the DisplayDarkMode.
Else you need to find out if the service is registered in the DI for this tenant which seems odd from a View.

Reply:
I was thinking about that, but if we have an extension method for `DisplayAsync()` it will be nice instead of adding a settings or add check everywhere we need to render the shape